inference-engine/ie_bridges/python/sample/speech_sample/speech_sample.py

```diff
@@ -0,0 +1,390 @@
+        devices[0] = 'GNA'
+
+        plugin_config['GNA_DEVICE_MODE'] = gna_device_mode
+        plugin_config['GNA_PRECISION'] = f'I{args.quantization_bits}'
+
+        # Get a GNA scale factor
+        if args.import_gna_model:
+            log.info(f'Using scale factor from the imported GNA model: {args.import_gna_model}')
+        else:
+            utterances = read_utterance_file(args.input.split(',')[0])
+            key = sorted(utterances)[0]
+            scale_factor = get_scale_factor(utterances[key])
+            log.info(f'Using scale factor of {scale_factor} calculated from first utterance.')
+
+            plugin_config['GNA_SCALE_FACTOR'] = str(scale_factor)
+
+        if args.export_embedded_gna_model:
+            plugin_config['GNA_FIRMWARE_MODEL_IMAGE'] = args.export_embedded_gna_model
+            plugin_config['GNA_FIRMWARE_MODEL_IMAGE_GENERATION'] = args.embedded_gna_configuration
+
+    device_str = f'HETERO:{",".join(devices)}' if 'HETERO' in args.device else devices[0]
+
+    log.info('Loading the model to the plugin')
+    if args.model:
+        exec_net = ie.load_network(net, device_str, plugin_config)
+    else:
+        exec_net = ie.import_network(args.import_gna_model, device_str, plugin_config)
+        input_blobs = [next(iter(exec_net.input_info))]
+        output_blobs = [list(exec_net.outputs.keys())[-1]]
+
+    if args.input:
+        input_files = re.split(', |,', args.input)
+
+        if len(input_blobs) != len(input_files):
+            log.error(f'Number of network inputs ({len(input_blobs)}) is not equal '
+                      f'to number of ark files ({len(input_files)})')
+            sys.exit(-3)
+
+    if args.reference:
+        reference_files = re.split(', |,', args.reference)
+
+        if len(output_blobs) != len(reference_files):
+            log.error('The number of reference files is not equal to the number of network outputs.')
+            sys.exit(-5)
+
+    if args.output:
+        output_files = re.split(', |,', args.output)
+
+        if len(output_blobs) != len(output_files):
+            log.error('The number of output files is not equal to the number of network outputs.')
+            sys.exit(-6)
+
+    if args.export_gna_model:
+        log.info(f'Writing GNA Model to {args.export_gna_model}')
+        exec_net.export(args.export_gna_model)
+        return 0
+
+    # TODO: Find a model that applicable for -we and -we_gen options
+    if args.export_embedded_gna_model:
+        log.info(f'Exported GNA embedded model to file {args.export_embedded_gna_model}')
+        log.info(f'GNA embedded model export done for GNA generation {args.embedded_gna_configuration}')
+        return 0
+
+# ---------------------------Step 5. Create infer request--------------------------------------------------------------
+# load_network() method of the IECore class with a specified number of requests (default 1) returns an ExecutableNetwork
```

Comment:
The default number of requests is plugin-specific and configuration-specific. For example, for MYRIAD it is not true (since we are mentioning this device in README). As far as I see currently only 1-st request is used in infer, I propose to set a number of requests directly to 1. Otherwise, other requests would not be used, but recoursed would be allocated.